gradle/native-build.gradle

```diff
@@ -149,7 +149,6 @@ ext.nativeBuild = { buildDir, projectDir, name, os ->
 
             def javacArgs = ["-d", dalvikOutput,
                              "-source", "1.7", "-target", "1.7",
-                             "-cp", "$projectDir/src/main/native/$os/dalvik/",
```

Comment:
This doesnt break when there are many java files in that directory?

Reply:
I don't think we need this. I copied it from [Substrate](https://github.com/gluonhq/substrate/blob/master/src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java#L395), where we have something similar, but probably that was expecting .class files, not source files?